include/win32/windows.h

```diff
@@ -56,10 +56,23 @@ extern "C" {
 #define DECLSPEC_ALIGN(x)   __declspec(align(x))
 
 /* Basic Defines: */
-#define NTAPI __stdcall
-#define WINAPI __stdcall
-#define APIENTRY __stdcall
-#define CALLBACK __stdcall
+#ifdef _WIN32
+
+    #define NTAPI __stdcall
```

Comment:
One of the main reasons for choosing and creating this alternative `windows.h` is to reduce the macros that pollute the code.

This is working against the goal. 

Therefore, I'd recommend to stick with a single macro following the system reserved convention: `_STDCALL`.